test-suite/misc/PStringExtraction.out

```diff
@@ -0,0 +1,71 @@
+File "./misc/PStringExtraction.v", line 25, characters 0-23:
+Warning: The following axiom must be realized in the extracted code:
+ int.
+ [extraction-axiom-to-realize,extraction,default]
+
+type comparison =
+| Eq
+| Lt
+| Gt
+
+type int (* AXIOM TO BE REALIZED *)
```

Comment:
There is something wrong here. Perhaps the file `ExtrOCamlPString` should import `ExtrOCamlInt63`?

Reply:
Fixed.